content/plugins/current/main/bridge.md

```diff
@@ -94,6 +105,7 @@ If the bridge is missing, the plugin will create one on first use and, if gatewa
 * `enabledad` (boolean, optional): enables duplicate address detection for the container side veth. Defaults to false.
 * `macspoofchk` (boolean, optional): Enables mac spoof check, limiting the traffic originating from the container to the mac address of the interface. Defaults to false.
 * `disableContainerInterface` (boolean, optional): Set the container interface (veth peer inside the container netns) state down. When enabled, IPAM cannot be used.
+* `portIsolation` (boolean, optional): Set isolation on the host interface (veth peer in root netns). Defaults to false.
```

Comment:
Can you add one more sentence? Something like "When enabled, containers can communicate only with the host"